SQL/New_patches/2019_07_05_Add_Language_feature.sql

```diff
@@ -0,0 +1,6 @@
+ALTER TABLE `session`
+    ADD COLUMN `languageID` integer unsigned DEFAULT NULL,
+    ADD CONSTRAINT `FK_session_4` FOREIGN KEY (`languageID`) REFERENCES `language` (`language_id`);
+
+INSERT INTO language (language_code, language_label) VALUES
+    ('fr-CA', 'French');
```

Comment:
The patch doesn't match the default schema change.

Reply:
en-CA is already part of the schema and presumably already in the db - is that wrong? Shoudl en-CA appear in the patch as well?

Reply:
Oh, I see. I misread the default schema change. I don't see why this is adding fr-CA.

Reply:
should i remove it>

Reply:
yes